build.gradle

```diff
@@ -272,6 +272,7 @@ subprojects {
         implementation 'com.fasterxml.jackson.core:jackson-annotations'
         implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
         implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
+        implementation 'com.jayway.jsonpath:json-path:2.7.0'
```

Comment:
if we make this package private then we might actually just put it in the commons dependencies instead of the deps in the root of the project so that only commons has access to it.